modules/openxr/scene/openxr_hand.h

```diff
@@ -86,10 +98,14 @@ class OpenXRHand : public Node3D {
 	void set_hand_skeleton(const NodePath &p_hand_skeleton);
 	NodePath get_hand_skeleton() const;
 
+	void set_skeleton_rig(const SkeletonRig p_skeleton_rig);
```

Comment:
```suggestion
	void set_skeleton_rig(SkeletonRig p_skeleton_rig);
```
We don't normally add `const` to simple types, it's been done in the past but we try to avoid it in new code

Reply:
Good catch - it was pinging my ReSharper but I erroneously chose to keep with the old pattern. This is now fixed and I also removed the unnecessary const from the other two simple-type setters.